docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,261 @@
+# Migrate from Pipedream v1
+
+The newest version of the Pipedream workflow introduces features like creating code steps in [Python](/code/python), [Bash](/code/bash) and [Go](/code/go).
+
+It also brings the [Node.js](/code/nodes) steps closer to our [Component API](/components/) which powers sources & actions in the platform.
+
+But this also means some of our features have changed. Here's what you need to know.
+
+## New Builder Overview
+
+Fundamentally, the new version of the workflow builder gives you the same abilities to build, test and deploy your workflows. However, there are some differences that change up your normal flow.
+
+### Editing vs Inspecting
+
+When you first open a deployed workflow, you're presented with the **inspector** version of the workflow. In this view you can see logs of past events, and select them to see the results of each step in the workflow.
```

Comment:
Inspector should be capitalized.

I'd recommend noting how users switch between Inspect and Build / Edit mode, e.g. maybe talk about the ellipsis on the top-right corner and include a screenshot or something.